contracts/callcenter/src/lib.rs

```diff
@@ -92,6 +92,11 @@ impl Callcenter {
         call_with_limit(contract, "spend", &(), points_limit)?;
         Ok(())
     }
+
+    /// Just panic.
+    pub fn panik(&self) {
```

Comment:
```suggestion
    pub fn panic(&self) {
```

Reply:
But I like to [`panik`](https://i.pinimg.com/originals/bd/ce/25/bdce2580f547211a9bbf43d04a9b50ff.jpg)